examples/advanced_examples/modified_express_audio.py

```diff
@@ -34,19 +36,25 @@ def _sine_waveform(length, amplitude=1):
             yield int(amplitude * max_amplitude * (math.sin(angle) + 1))
```

Comment:
You might see if using a regular loop is noticeably faster than using `yield`. I'm not sure how much difference it would make due to the cost of `sin()`. Another idea is to have some precomputed `sin()` values. (Maybe you discussed that in the Monday meeting? I was not following closely.)

Reply:
Thanks, Dan. I’m thinking Kattni (if it was she) used yield to avoid having two copies of the sample in memory at the same time, since it is loaded into an array. In any case, these sample-generating methods will no longer be the bottleneck.